nototools/noto_cmap_reqs.py

```diff
@@ -435,7 +451,7 @@ def _reassign_common_by_block(cmap_ops):
     'Enclosed Alphanumerics': 'Zsym',
     'Box Drawing': 'MONO',
     'Block Elements': 'MONO',
-    'Geometric Shapes': 'Zsym',
+    'Geometric Shapes': 'SYM2', # change
```

Comment:
Just wondering (not proposing any change): Can you think of any scenario in which using 'SYM2' (not valid per ISO 15924) will bite us in the future  ? Extremely unlikely because it's just our internal code and 'SYM2' is infinitely more "readable" than "Qaaa' (picked from the private use range).